mlrun/utils/model_monitoring.py

```diff
@@ -156,6 +156,53 @@ def get_connection_string(project: str = None):
         )
 
 
+def get_stream_path(project: str = None):
+    """Get stream path from the project secret. If wasn't set, take it from the system configurations"""
+
+    if is_running_as_api():
+
+        # Running on API server side
+        import mlrun.api.crud.secrets
+        import mlrun.api.schemas
+
+        stream_uri = mlrun.api.crud.secrets.Secrets().get_project_secret(
+            project=project,
+            provider=mlrun.api.schemas.secret.SecretProviderName.kubernetes,
+            allow_secrets_from_k8s=True,
+            secret_key=model_monitoring_constants.ProjectSecretKeys.STREAM_PATH,
+        ) or mlrun.mlconf.get_file_target_path(
+            project=project,
+            kind=model_monitoring_constants.FileTargetKind.STREAM,
+            target=model_monitoring_constants.FileTargetKind.STREAM,
+        )
+
+    else:
+
+        import mlrun
+
+        stream_uri = mlrun.get_secret_or_env(
+            model_monitoring_constants.ProjectSecretKeys.STREAM_PATH
+        ) or mlrun.mlconf.get_file_target_path(
+            project=project,
+            kind=model_monitoring_constants.FileTargetKind.STREAM,
+            target=model_monitoring_constants.FileTargetKind.STREAM,
+        )
+
+    if (
+        stream_uri.startswith("kafka://")
+        and "topic=monitoring_stream_" not in stream_uri
+    ):
+        # Add topic to stream kafka uri
+        stream_uri += f"?topic=monitoring_stream_{project}"
```

Comment:
Do we always enforce adding a topic? What if the user doesn't want to use specific topics? Can't the user set the stream path to be a URL without topic?

Reply:
What we have talked about offline - we prevent the user from passing his own topic.